src/shared_modules/utils/rocksDBQueueCF.hpp

```diff
@@ -23,6 +23,10 @@
 #include <stdexcept>
 #include <string>
 
+constexpr auto ROCKSDB_CF_ID_PADDING {10};
+constexpr auto ROCKSDB_CF_NUMBER_PADDING {10};
```

Comment:
After discussing this in the daily, we are going to only add padding to the ID (as this follows the lexicographical order), and knowing that agents can have 8 digits in their ID (https://github.com/wazuh/wazuh/issues/6042)

We are going to pad to a size of 30